### PR TITLE
150 hours of testing (fixes master)

### PR DESCRIPTION
**Pull request by silicons**
Thanks, Robustin!
Very ROBUST code you got there!

### CODE REVIEW

**Comment by kevinz000** [comment]
@Robustin you added a line that relies on G existing which doesn't in your change.

File: code/game/gamemodes/gang/gang.dm [hunk 1]
```diff
@@ -313,6 +313,7 @@ GLOBAL_LIST_INIT(gang_colors_pool, list("red","orange","yellow","green","blue","
 				G.domination(0.5)
 			priority_announce("Multiple station takeover attempts have made simultaneously. Conflicting takeover attempts appears to have restarted.","Network Alert")
 		else
+			var/datum/gang/G = winners[1]
```
Comment (kevinz000) [inline code/game/gamemodes/gang/gang.dm]: done, set to TRUE.

**Comment by kevinz000** [comment]
my shitposting is justified by the fact that you would of found out if you opened dream maker and hit compile
i'm actually wondering why travis never ran on a pr that was open for 2 days =/

**Comment by Robustin** [comment]
To be fair I did title my PR "probably fixes" which should be a red flag to anyone reading it that I didn't bother testing it.

**Comment by kevinz000** [comment]
Don't worry robustin we still love you bad luck on the headmin elections <\3

**Pull request merged**